### PR TITLE
Blueprint updates

### PR DESCRIPTION
**Pull request by jvega190**
- Fix logo image rendering
- Disable image spinning when editMode is on
- Add prettier script
- Update image datasources for image-picker and rte controls

### CODE REVIEW

**Comment by jvega190** [comment]
@rart ^

**Review by rart — APPROVED** [review]

**Review by sumerjabri — CHANGES_REQUESTED** [review]
Please update the craftercms-plugin.yaml to update the blueprint version and the compatibility matrix.

**Review by rart — APPROVED** [review]

File: craftercms-plugin.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ plugin:
   version:
     major: 1
     minor: 0
-    patch: 1
+    patch: 2
```
Comment (sumerjabri) [inline craftercms-plugin.yaml:18]: Is this not compatible with 4.1.3-SNAPSHOT, 4.2.0-SNAPSHOT?

**Review by sumerjabri — CHANGES_REQUESTED** [review]
CC @rart

**Review by sumerjabri — APPROVED** [review]

**Pull request merged**